### PR TITLE
update to use btcsnap fork

### PR DESCRIPTION
**Pull request by gregdhill**

### CODE REVIEW

File: app/src/utils/btcsnap-signer.ts [hunk 1]
```diff
@@ -81,15 +84,19 @@ export class BtcSnapSigner implements RemoteSigner {
 
   async getPublicKey(): Promise<string> {
     const network = await this.getNetwork();
+    if (network === bitcoin) {
+      await updateNetworkInSnap(BitcoinNetwork.Test)
```
Comment (bvotteler) [inline app/src/utils/btcsnap-signer.ts:88]: Is this a typo, or on purpose to force testnet for the demo?

It looks odd to query the snap in the previous statement (inside `this.getNetwork()`) to then set the network in the snap. I'm not sure what the intention here is.

File: app/src/utils/btcsnap-utils.ts [hunk 2]
```diff
@@ -370,6 +370,39 @@ export async function signPsbt(base64Psbt: string, network: BitcoinNetwork, scri
...
+): Promise<string> {
+  const networkParams = network === BitcoinNetwork.Main ? 'main' : 'test';
+
+  try {
+    return (await ethereum.request({
+      method: 'wallet_invokeSnap',
+      params: {
+        snapId,
+        request: {
+          method: 'btc_signInput',
```
Comment (bvotteler) [inline app/src/utils/btcsnap-utils.ts:388]: Oooh, great find!

Comment (gregdhill) [reply app/src/utils/btcsnap-utils.ts:388]: This is a change I made to the snap

File: app/src/utils/btcsnap-signer.ts (hunk 1 above)
Comment (gregdhill) [reply app/src/utils/btcsnap-signer.ts:88]: Removed I just had it to force the network

**Pull request merged**